packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -33,39 +34,56 @@ class FlatTable<T> extends StatefulWidget {
   _FlatTableState<T> createState() => _FlatTableState<T>();
 }
 
-class _FlatTableState<T> extends NestingTableState<T, FlatTable<T>> {
-  @override
-  bool get startAtBottom => true;
+class _FlatTableState<T> extends State<FlatTable<T>> {
+  List<double> columnWidths = [];
```

Comment:
why is this initialized when it is reset by initState?

Reply:
Done.